package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "inclusivity-slack-inviter",
+  "version": "0.0.0",
+  "description": "Automated Slack invitations for the Node.js Inclusivity WG",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
Ok. I've actually got a few questions around this:
* Do we use "Copyright Node.js Inclusivity Contributors"? (inclusivity does, I tend to prefer these forms that are correct and never need to be updated)
* Do we include a LICENSE file? (inclusivity doesn't, but I think we should)
* Do we use an implicit DCO? (inclusivity does, I'll refrain from commenting on this)
* Is there some document that covers legal aspects for WGs?